### ISSUE TITLE
The shape of homogenization RVE 

### ISSUE THREAD

**Issue by CebFu** [MeshFEM/MeshFEM#10]
Hi, everyone. It said in the Readme description that you can run periodic homogenization on a periodic microstructure (2D or 3D) in a unit cell (square in 2D, cube in 3D) . I want to know whether it applies to general rectangular and cuboid geometry

**Comment by jpanetta** [comment]
Yes, it should just work for rectangular unit cells. It also should work for more general unit cell shapes (e.g., parallelograms and hexagons), but for these you need to [manually specify the periodically identified nodes](https://github.com/MeshFEM/MeshFEM/blob/5c92534a2d6db5c8007b6e6e25a84ce84c7d3d84/src/lib/MeshFEM/BoundaryConditions.hh#L567) and the volume of the RVE.

**Comment by CebFu** [comment]
@jpanetta Thank you for your reply.
